rust/src/container.rs

```diff
@@ -387,6 +398,20 @@ pub fn container_encapsulate(args: Vec<String>) -> CxxResult<()> {
         })
         .collect::<Result<_>>()?;
 
+    let package_structure = match &opt.ocimeta_json {
+        Some(ocimeta_json) => {
+            let path = std::path::Path::new(ocimeta_json);
+            if path.is_file() {
```

Comment:
Why this check?  I think we should just error out if the user passes us e.g. a directory.

Reply:
According the [documentation](https://doc.rust-lang.org/std/path/struct.Path.html#method.is_file), it will also error out if file doesnt exist at all in the disk or a permission error

Reply:
If the user typos the filename, I think we *do* want to error out and not just silently ignore it, right?

Reply:
Right makes sense. Corrected that